pkg/filter/llm/proxy/filter.go

```diff
@@ -150,14 +149,14 @@ func (factory *FilterFactory) PrepareFilterChain(ctx *contexthttp.HttpContext, c
 func (f *Filter) Decode(hc *contexthttp.HttpContext) filter.FilterStatus {
 	rEntry := hc.GetRouteEntry()
 	if rEntry == nil {
-		sendJSONError(hc, http.StatusBadRequest, "no route entry found for request")
+		contexthttp.BadRequest.SendError(hc, fmt.Errorf("no route entry found for request"))
```

Comment:
if the format func is not used when creating a new err, use `errors.New` instead of `fmt.Errorf`. `fmt.Errorf` only used when need format

```suggestion
		contexthttp.BadRequest.SendError(hc, errors.New("no route entry found for request"))
```